core/modules/simpletest/backdrop_web_test_case.php

```diff
@@ -1354,7 +1354,7 @@ protected function backdropCreateRole(array $permissions, $name = NULL) {
       $name = $this->randomName();
     }
 
-    // Check the all the permissions strings are valid.
+    // Check that all the permission strings are valid.
     if (!$this->checkPermissions($permissions)) {
```

Comment:
If I change this bit here to this, those nasty "Invalid permission" failures go away and everything is green 🤔 

```suggestion
    if (!$this->checkPermissions($permissions, TRUE)) {
```

I've raised https://github.com/backdrop/backdrop-issues/issues/5354 to discuss this further.